models/generators.py

```diff
@@ -180,7 +158,7 @@ def loss(self, predictions_on_fake, mean, log_sigma):
         """ Only calculate the loss based on the discriminator
             predictions for the images generated by this model.
         """
-        kl_coeff = 1
+        kl_coeff = 2
```

Comment:
Is this change in the paper? Just checking because I don't understand

Reply:
yeah, paper